tests/test_benchmark.py

```diff
@@ -811,7 +811,7 @@ def test_life(self):
     src = open(path_from_root('tests', 'life.c'), 'r').read()
     self.do_benchmark('life', src, '''--------------------------------''', shared_args=['-std=c99'], force_c=True)
 
-  def test_linpack(self):
+  def test_zzz_linpack(self):
```

Comment:
Do we still need/want this magic syntax.  Is the `@slow` annotation that we elsewhere good enough?  Or maybe its serving a different purpose here?

Reply:
I think `@slow` filters things out, while this sorts them to the end, which is nice when viewing the results. Is there a nicer way to sort maybe?